test/component/deceased/testAlias.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const TestWrapper = require('test/util/TestWrapper');
+const DeceasedOtherNames = require('app/steps/ui/deceased/otherNames/index');
+const EndOfJourney = require('app/steps/ui/endjourney/index');
+const testHelpBlockContent = require('test/component/common/testHelpBlockContent.js');
+
+describe('deceased-alias', () => {
+    let testWrapper;
+    const expectedNextUrlForDeceasedOtherNames = DeceasedOtherNames.getUrl();
+    const expectedNextUrlForEndOfJourney = EndOfJourney.getUrl();
+
+    beforeEach(() => {
+        testWrapper = new TestWrapper('DeceasedAlias');
+    });
+
+    afterEach(() => {
+        testWrapper.destroy();
+    });
+
+    describe('Verify Content, Errors and Redirection', () => {
+        testHelpBlockContent.runTest('DeceasedAlias');
+
+        it('test right content loaded on the page', (done) => {
+            const sessionData = {
+                deceased: {
+                    firstName: 'John',
+                    lastName: 'Doe'
+                }
+            };
+            const excludeContent = ['theDeceased'];
+
+            testWrapper.agent.post('/prepare-session/form')
+                .send(sessionData)
+                .end(() => {
+
+                    const contentData = {deceasedName: 'John Doe'};
+
+                    testWrapper.testContent(done, excludeContent, contentData);
+
+                });
+        });
+
+        it('test alias schema validation when no data is entered', (done) => {
+            const data = {};
+
+            testWrapper.testErrors(done, data, 'required', []);
+        });
+
+        it(`test it redirects to deceased other names page: ${expectedNextUrlForDeceasedOtherNames}`, (done) => {
+            const data = {
+                alias: 'Yes'
+            };
+            testWrapper.testRedirect(done, data, expectedNextUrlForDeceasedOtherNames);
+        });
+
+        it(`test it redirects to deceased other names page: ${expectedNextUrlForEndOfJourney}`, (done) => {
```

Comment:
Update the test description

Reply:
changes description to:
 test it redirects to end of journey page: